packages/ui/src/hooks/entities.ts

```diff
@@ -108,3 +123,15 @@ function getEntity(rawEntity: unknown): BaseCamelEntity | BaseVisualCamelEntity
 
   return rawEntity as BaseCamelEntity;
 }
+
+function getEntityType(entity: unknown): EntityType {
+  if (isKameletBinding(entity)) {
+    return EntityType.KameletBinding;
+  }
+
+  if (isCamelRoute(entity)) {
+    return EntityType.Route;
+  }
+  if (isPipe(entity)) return EntityType.Pipe;
+  return EntityType.Pipe;
+}
```

Comment:
I think that in this case, we need to set the `Camel YAML DSL` schema as default, and only change the schema for:

* KameletBindings
* Pipes
* Kamelet

So then the code should read more or less like:

```
  if (isKameletBinding(entity)) return EntityType.KameletBinding;

  if (isKamelet(entity)) return EntityType.Kamelet;

  if (isPipe(entity)) return EntityType.Pipe;

  if (isIntegration(entity)) return EntityType.Integration;

  // After reaching this point, it should be enough to set the `Camel YAML DSL` schema
  return EntityType.Route;
```

Reply:
done